src/util.ts

```diff
@@ -658,6 +658,28 @@ export async function handleFetch(request: string, options?: RequestInit) {
   return object;
 }
 
+/**
+ * Execute fetch and return object response, log if known error thrown, otherwise rethrow error.
+ *
+ * @param request - The request information.
+ * @param options - The fetch options.
+ * @returns The fetch response JSON data or undefined (if error occurs).
+ */
+export async function fetchWithErrorHandling(
+  request: string,
+  options?: RequestInit,
+) {
+  let object;
```

Comment:
Was this meant to be `response`? This is always `undefined` right now.

Reply:
I don't follow why you didn't return on line 675. This variable doesn't seem needed at all.

Reply:
yes on both counts... 🙄